save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -188,6 +190,12 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
                                 val trdi = getDebugInfoFor(te)
                                 if (trdi.ok) {
                                     cellProps.row.original.asDynamic().debugInfo = trdi.decodeFromJsonString<TestResultDebugInfo>()
+                                } else {
+                                    val trei = getExecutionInfoFor(te)
```

Comment:
This information should be displayed not only when debug info is unavailable, but also somewhere in the general description of the execution. I think it  would make more sense to see error text right away if tests didn't even begin.

Reply:
fixed